src/vm/node_modules/VM.js

```diff
@@ -13441,7 +13441,11 @@ exports.delete_snapshot = function (uuid, snapname, options, callback)
                         + 'unmounting the snapshot: ' + e.message);
                     // we treat an error here as fatal only if the error
                     // was something other than 'not mounted'
-                    if (!stderr.match(/ not mounted/)) {
+                    //                       or 'no such file or directory'
```

Comment:
Not sure why this is indented as such - I'd prefer to see the comment in proper sentence format.

Reply:
An earlier iteration combined both errors in a single regex, this was an attempt to keep the list of conditions easily readable.
Since the conditions are now checked separately, the indenting is no longer needed.